docs/book/src/reference/owner_references.md

```diff
@@ -0,0 +1,79 @@
+# Owner References
+
+
```

Comment:
Should we add a note somewhere that we enforce the ownerRefs are reconciled and re-reconciled directly via verifications at the end of the quickstart e2e test and we would recommend infra providers to do the same?